widgetastic_manageiq/__init__.py

```diff
@@ -1821,8 +1822,9 @@ def button_names(self):
     def selected(self):
         names = self.button_names
         for name in names:
-            if 'ng-valid-parse' in self.browser.classes('.//input[@type="radio"]',
-                                                        parent=self._get_button(name)):
+            bttn = self.browser.element('.//input[@type="radio"]', parent=self._get_button(name))
+            if ('ng-valid-parse' in self.browser.classes(bttn) or
+               bttn.get_attribute('checked') is not None):
```

Comment:
**REQUIRED**: Use double indent in here instead of indenting by 3 spaces.

Reply:
done